services/movie_session.py

```diff
@@ -42,3 +42,15 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    list_of_row_seat = list(
+        MovieSession.objects.filter(id=movie_session_id).values(
+            "ticket__row", "ticket__seat"
+        )
+    )
+    return [
+        {"row": c["ticket__row"], "seat": c["ticket__seat"]}
+        for c in list_of_row_seat
+    ]
```

Comment:
The `get_taken_seats` function correctly retrieves and formats the list of taken seats for a given movie session. Ensure that the `movie_session_id` is valid before calling this function to avoid potential errors.